test/tf/azurerm/main.tf

```diff
@@ -3,11 +3,11 @@ provider "azurerm" {
 }
 
 provider "random" {
-  version = "~> 1.2"
+  version = "~> 2.0.0"
 }
 
 variable "prefix" {
-  default = "go-discover"
+  default = "go-discover-azurerm"
```

Comment:
Is there any reason we shouldn't add a random string to this prefix while we're at it? Seems like that would resolve any other collision potential.

Reply:
That could work but we would have to have the random string somehow end up in: https://github.com/hashicorp/go-discover/blob/24a1d7891707298d821fbed19f96305b544ccb1c/provider/azure/azure_discover_test.go#L18 and likewise for the other one.

Reply:
Ah yeah didn't realize it was part of those parameters. Cool lets leave as-is then.